code/triangle/Model/src/main/java/ru/unn/agile/triangle/Triangle.java

```diff
@@ -22,6 +25,12 @@ public Triangle(final Point a, final Point b, final Point c) {
         this.a = a;
         this.b = b;
         this.c = c;
+
+        this.numberFormat = new DecimalFormat();
+        DecimalFormatSymbols decimalFormatSymbols = numberFormat.getDecimalFormatSymbols();
+        decimalFormatSymbols.setDecimalSeparator('.');
+        numberFormat.setDecimalFormatSymbols(decimalFormatSymbols);
+        numberFormat.setMaximumFractionDigits(2);
```

Comment:
Maybe it would be better to create separate class/only method which will be creating DecimalFormat and setting parameters? Because you use this values 2 times.